modules/openapi-generator/src/main/resources/typescript-aurelia/package.json.mustache

```diff
@@ -1,21 +1,23 @@
 {
-  "name": "{{#npmName}}{{{npmName}}}{{/npmName}}{{^npmName}}typescript-fetch-api{{/npmName}}",
+  "name": "{{#npmName}}{{{npmName}}}{{/npmName}}{{^npmName}}typescript-aurelia-api{{/npmName}}",
   "version": "{{#npmVersion}}{{{npmVersion}}}{{/npmVersion}}{{^npmVersion}}0.0.0{{/npmVersion}}",
   "license": "Unlicense",
-  "main": "./dist/api.js",
-  "browser": "./dist/api.js",
-  "typings": "./dist/api.d.ts",
+  "main": "./dist/Api.js",
+  "browser": "./dist/Api.js",
+  "typings": "./dist/Api.d.ts",
   "dependencies": {
     {{^supportsES6}}"core-js": "^2.4.0",
-    {{/supportsES6}}"isomorphic-fetch": "^2.2.1"
+    {{/supportsES6}}"isomorphic-fetch": "^2.2.1",
+    "aurelia-framework": "^1.3.1",
+    "aurelia-http-client": "^1.3.0"
   },
   "scripts" : {
-    "prepublish" : "typings install && tsc", 
-    "test": "tslint api.ts"
+    "build": "tsc --outDir dist/",
+    "prepare": "npm run build",
+    "test": "tslint Api.ts"
   },
   "devDependencies": {
     "tslint": "^3.15.1",
-    "typescript": "^1.8.10",
-    "typings": "^1.0.4"
+    "typescript": "^2.4"
```

Comment:
is typescript 3.x also allowed?

Reply:
it seem to work with latest typescript version so i'll update the PR with it